actions/admin/settings/150.mail.php

```diff
@@ -162,7 +182,27 @@
 					'string_emptyallowed' => true,
 					'save_method' => 'storeSettingField',
 					'advanced_mode' => true
-				]
+				],
+                'system_mta_conf_dir' => [
+                    'label' => lng('serversettings.mta_conf_dir'),
+                    'settinggroup' => 'system',
+                    'varname' => 'mta_conf_dir',
+                    'type' => 'text',
+                    'string_type' => 'filedir',
```

Comment:
Same as mda conf dir, the type should be `confdir` and the default value should end with a `/`